gamestonk_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -622,3 +629,193 @@ def call_swaps(self, other_args: List[str]):
                 descend=ns_parser.descend,
                 export=ns_parser.export,
             )
+
+    @try_except
```

Comment:
You can remove this line. kudos to @colin99d
Had to remove that to be able to run the PR

Reply:
You mean to remove @try_except line ?

Reply:
yea, @colin99d refactored the menus so we don't need to add that in every function